### PR TITLE
Orie Steele review

### PR DESCRIPTION
**Pull request by marco-tiloca-sics**
This PR addresses the review from Orie Steele archived at https://mailarchive.ietf.org/arch/msg/ace/xyEYml0cSYVZWURk_yh8MsG5YBs/

### CODE REVIEW

File: draft-ietf-ace-revoked-token-notification.md [hunk 1]
```diff
@@ -215,6 +219,51 @@ At a high level, the steps of this protocol are as follows.
...
+
+    In turn, the resulting tagged data item MUST be tagged by using the CWT CBOR tag with tag number 61 (see {{Section 6 of RFC8392}}). After that, the resulting data item MUST NOT be further tagged.
+
+    Encoding of the tag numbers MUST be done using definite lengths, and the length of the encoded tag numbers MUST be the minimum possible length. This means that the tag number 16 is encoded as 0xd0 and not as 0xd810.
+
+    The example in {{fig-cwt-cose-encrypt0}} shows a CWT that uses the COSE object COSE_Encrypt0 (see {{Section 5.2 of RFC9052}}).
+
+* If the access token is a JWT, the following applies.
+
+  - If the access token is specifically a JWS {{RFC7515}} that uses the JWS JSON Serialization, it MUST NOT include the JWS Unprotected Header.
```
Comment (OR13) [inline draft-ietf-ace-revoked-token-notification.md]: A JWT is never allowed to be in JSON serialization.

> JWTs are always represented using the JWS Compact Serialization or the JWE Compact Serialization.

- https://datatracker.ietf.org/doc/html/rfc7519#section-1

Never the less, there are known cases, such as JAdES where JSON serialization is used:

- https://www.etsi.org/deliver/etsi_ts/119100_119199/11918201/01.01.01_60/ts_11918201v010101p.pdf

There is also SD-JWT, which also supports JSON Serialization, adding to the confusion.

You might be able to use "if the token is Compact Serialization such as JWS Compact Serialization or the JWE Compact Serialization...".

This comment in a nit.

File: draft-ietf-ace-revoked-token-notification.md [hunk 2]
```diff
@@ -851,6 +912,8 @@ Once registered at the AS, the administrator or registered device can send a GET
 
 In case the request is successfully processed, the AS replies with a response specifying the CoAP response code 2.05 (Content). In particular, if the AS supports diff queries but not the "Cursor" extension (see {{sec-trl-endpoint-supporting-diff-queries}} and {{sec-trl-endpoint-supporting-cursor}}), then the payload of the response is formatted as defined in {{ssec-trl-full-query}} or in {{ssec-trl-diff-query}}, in case the GET request has yielded the execution of a full query or of a diff query of the TRL, respectively. Instead, if the AS supports both diff queries and the related "Cursor" extension, then the payload of the response is formatted as defined in {{sec-using-cursor}}.
 
+In case a requester does not receive a response from the TRL endpoint or it receives an error response from the TRL endpoint, the requester does not make any assumption or draw any conclusion regarding the revocation or expiration of its pertaining access tokens.
```
Comment (OR13) [inline draft-ietf-ace-revoked-token-notification.md]: You could mention a retry policy here.

**Review by OR13 — APPROVED** [review]

File: draft-ietf-ace-revoked-token-notification.md (hunk 2 above)
Comment (marco-tiloca-sics) [reply draft-ietf-ace-revoked-token-notification.md]: Now addressed at: https://github.com/ace-wg/ace-revoked-token-notification/pull/13/commits/0db3c7be32172151dc139eb8255a2e880cffa432

File: draft-ietf-ace-revoked-token-notification.md (hunk 1 above)
Comment (marco-tiloca-sics) [reply draft-ietf-ace-revoked-token-notification.md]: Thanks for this comment! It is now addressed at: https://github.com/ace-wg/ace-revoked-token-notification/pull/13/commits/ab91b71c652ff37ee96c8632071103ee4b8f4232

The JWS Compact Serialization from RFC 7515 and the JWE Compact Serialization from RFC 7516 do not admit unprotected headers altogether (see Section 3.1 of both RFCs).

This means that JWTs as defined in RFC 7519 are basically just fine. The explicit requirements on avoiding unprotected headers are needed only for possible other access tokens that use the JWS/JWE JSON Serialization.

Now the text in Sections 3 and 11.1 reflects that.

**Pull request merged**